assets/src/edit-story/components/canvas/useInsertTextSet.js

```diff
@@ -14,18 +14,21 @@
  * limitations under the License.
  */
 
+/**
+ * External dependencies
+ */
+import { useCallback } from 'react';
+
 /**
  * Internal dependencies
  */
-import useBatchingCallback from '../../../../../utils/useBatchingCallback';
-import objectWithout from '../../../../../utils/objectWithout';
-import useLibrary from '../../../useLibrary';
-import { useStory } from '../../../../../app/story';
+import useBatchingCallback from '../../utils/useBatchingCallback';
+import objectWithout from '../../utils/objectWithout';
+import { useStory } from '../../app/story';
+import useInsertElement from './useInsertElement';
 
 function useInsertTextSet() {
-  const { insertElement } = useLibrary((state) => ({
-    insertElement: state.actions.insertElement,
-  }));
+  const insertElement = useInsertElement();
```

Comment:
I needed access to `useInsertTextSet` from within the `canvasElementDropzone.js` so I could track the drop data of the text set.  The problem was that `useInsertTextSet` (in its original location) had access to the `useLibrary` context, however  `canvasElementDropzone.js` does not... instead of re-working everything so `canvasElementDropzone.js` would have access to the library context, I simply moved `useInsertTextSet.js` to the `canvas` folder and referenced `useInsertElement` directly instead of through the library.